extension.js

```diff
@@ -169,12 +171,21 @@ function assertNextJSApp(componentPath) {
  */
 async function executeCommand(commandInput, componentPath) {
 	const [command, ...args] = shellQuote.parse(commandInput);
-	const cp = child_process.spawn(command, args, {
+	const env = { ...process.env, PATH: `${process.env.PATH}:${componentPath}/node_modules/.bin` };
+	let cp = child_process.spawn('which', [command], { cwd: componentPath, env, stdio: 'ignore' });
```

Comment:
If the command can't be found, doesn't the ENOENT error code unambiguously indicate that? Is it necessary to do a `which` first or can we just listen for errors on the main spawn call and translate ENOENT to the appropriate error message.

Reply:
Good point